Cubical/Categories/Displayed/LeftAdjointToReindex.agda

```diff
@@ -0,0 +1,107 @@
+{-# OPTIONS --safe #-}
+--
+module Cubical.Categories.Displayed.LeftAdjointToReindex where
```

Comment:
This should be `Cubical.Categories.Displayed.Constructions.LeftAdjointToReindex`